src/taker/routines.rs

```diff
@@ -92,23 +92,12 @@ pub fn handshake_maker(socket: &mut TcpStream) -> Result<(), TakerError> {
     }
 }
 
-/// Request signatures for sender side of the hop. Attempt once.
-pub(crate) fn req_sigs_for_sender_once<S: SwapCoin>(
-    socket: &mut TcpStream,
-    outgoing_swapcoins: &[S],
+fn build_contract_tx_info_for_sender<S: SwapCoin>(
     maker_multisig_nonces: &[SecretKey],
     maker_hashlock_nonces: &[SecretKey],
-    locktime: u16,
-) -> Result<ContractSigsForSender, TakerError> {
-    log::info!("Connecting to {}", socket.peer_addr()?);
-    handshake_maker(socket)?;
-    log::info!(
-        "===> Sending ReqContractSigsForSender to {}",
-        socket.peer_addr()?
-    );
-
-    // TODO: Take this construction out of function body.
```

Comment:
@mojoX911 This PR solves 2 TODOs

Reply:
The intention of this TODO is to take the message construction out of the `send_message` function body to increase readability. The TODO is already solved here in this function, but for some reason, I forgot to remove it. So it's redundant. 

It still applies in the `req_sigs_for_recvr_once` function where the message is constructed within the `send_message` function. That could be a valid PR.